datadogclient/datadog_client_test.go

```diff
@@ -44,8 +44,9 @@ var _ = Describe("DatadogClient", func() {
 			"test-deployment",
 			"dummy-ip",
 			time.Second,
-			1500,
+			2000,
 			gosteno.NewLogger("datadogclient test"),
```

Comment:
Note: Modified this value because adding additional tags was causing the tests to fail because the payload was too big